### PR TITLE
push projects to kafka

### PR DESCRIPTION
**Pull request by rakeshSgr**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -26,4 +26,5 @@ KAFKA_URL = "172.31.0.4:9092"
 KAFKA_GROUP_ID = "projects"                                                     // Kafka group id                                
 
 # SUBMISSION TOPIC
-SUBMISSION_TOPIC = "dev.sl.projects.submissions"                                // Kafka topic name for pushing projects submissions
\ No newline at end of file
+SUBMISSION_TOPIC = "dev.sl.projects.submissions"                                // Kafka topic name for pushing projects submissions
+IMPROVEMENT_PROJECT_SUBMISSION_TOPIC = "dev.sl.projects.imp_submissions"        // project submission topic
```
Comment (aks30) [inline .env.sample:30]: @rakeshSgr - Make it "dev.sl.projects.submissions"

Comment (aks30) [reply .env.sample:30]: @rakeshSgr the key can be "PROJECT_SUBMISSION_TOPIC"

File: module/userProjects/helper.js [hunk 2]
```diff
@@ -339,6 +341,8 @@ module.exports = class UserProjectsHelper {
                     }
                 }
 
+                await kafkaProducersHelper.pushImrovementProjectToKafka(projectUpdated);
```
Comment (aks30) [inline module/userProjects/helper.js:344]: @rakeshSgr let the function name be "pushProjectForReporting"

File: .env.sample (hunk 1 above)
Comment (rakeshSgr) [reply .env.sample]: resolved

File: module/userProjects/helper.js (hunk 2 above)
Comment (rakeshSgr) [reply module/userProjects/helper.js]: resolved

**Pull request merged**